frontend/src/assets/base.css

```diff
@@ -64,7 +64,7 @@ body {
   background: var(--color-background);
   transition: color 0.5s, background-color 0.5s;
   line-height: 1.6;
-  font-family: Inter, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family: Inter, -apple-system, BCSans, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
```

Comment:
Should BCSans not be at the front of the list?

https://www.w3schools.com/cssref/pr_font_font-family.php
`font-family: family-name|generic-family|initial|inherit;`
`family-name / generic-family: A prioritized list of font family names and/or generic family names`

Seems it is currently prioritizing a Google font, followed by a default apple font (if they exist)

Reply:
I followed the CHEFS example (as we talked about): https://github.com/bcgov/common-hosted-form-service/blob/50ddb54932e1bac626a270eea7d353c669f3ea33/app/frontend/src/assets/scss/style.scss#L41

But no problem, will put it at the front.